app/api/routes.py

```diff
@@ -205,7 +203,7 @@ def get_categories():
         print(e)
         category_list = []
     finally:
-        return standardize_response(category_list, None, "ok")
+        return standardize_response(payload=dict(data=category_list), status_code=200)
```

Comment:
Since the default for `status_code` is 200, I don't think we need that to be specified here